js/dimensions.js

```diff
@@ -136,8 +136,12 @@ $(document).ready(function() {
 
   $(document).on('change', '#dimension_image_upload', function() { Resize.readFile(this); });
   $(document).on('change', '#resize-select', function() { Resize.fillDimensionFields(this); });
-  $(document).on('click', '.preview-result', function() { Resize.showResult(); });
   $(document).on('click', '.submit-btn', function(event) { Resize.downloadableResult(); });
+  $(document).on('click', '.preview-result', '#user-width, #user-height', function() {
+    if (($("#user-height").val() <= 400 && $("#user-width").val() <= 400) && (Resize.originalHeight > 800 && Resize.originalWidth > 800)) {
```

Comment:
This code can be clearer. What does this mean:

    ($("#user-height").val() <= 400 && $("#user-width").val() <= 400)

Reply:
It's checking to see if the dimensions the user selects is less than or equal to 400x400 when cropping. Should I write a comment explaining that or write it another way?

Reply:
You get to decide that. I just want something that's unambiguous and easy to parse. Currently I look at this and it's not clear to my why these magic numbers were chosen or even exactly why this code is here. This needs to be clear now and in the future.